src/gwt/src/org/rstudio/studio/client/workbench/views/jobs/view/JobsPane.java

```diff
@@ -90,7 +90,7 @@ public void updateJob(int type, Job job)
             list_.addJob(job);
             
             // bring the pane to the front so the new job is visible
-            bringToFront();
+            if (job.show) bringToFront();
```

Comment:
We should also skip firing the selection event if we aren't showing the job right away -- otherwise it'll still switch to the job if you have the pane open which probably isn't desired.